### PR TITLE
telemetry test

### PR DESCRIPTION
**Pull request by Stojek139808**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -74,3 +74,73 @@ jobs:
         run: |
           aws s3 cp ${PROJECT}_${NEW_VERSION}.bin s3://dev-vitro-shard-applications/bumped-versions/
           aws s3 cp ${PROJECT}_${NEW_VERSION}_upgrade_package.swu s3://dev-vitro-core-sys-images/
+
+  build-telementry:
+    name: Build telementry test app
```
Comment (TomaszAIR) [inline .github/workflows/ci.yml:79]: @Stojek139808 should it not be `telemetry`?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -74,3 +74,73 @@ jobs:
...
+          path: "shard-v2-app-template"
+      - name: Get version
+        run: |
+          cd shard-v2-app-template
+          MAJOR=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 1p`
+          MINOR=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 2p`
+          PATCH=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 3p`
+          RC=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | awk '{print $3}' | sed -n 4p`
+          CURRENT_VERSION="$MAJOR.$MINOR.$PATCH.$RC"
+          PROJECT="telementry-test"
```
Comment (TomaszAIR) [inline .github/workflows/ci.yml:94]: Also here, telemetry, please check other appearances

File: Applications/telementry-test/main.cpp [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+            if(++blockCounter == 60){
+                type = STANDARD_BLOCK;
+                blockCounter = 0;
+            }
+            break;
+        case STANDARD_BLOCK:
+            sendBlock(iotBlock, type);
+            ThisThread::sleep_for(60s);
+            if(++blockCounter == 5){
+                type = STANDARD_BLOCK;
```
Comment (TomaszAIR) [inline Applications/telementry-test/main.cpp]: @Stojek139808 this should change to `DISCHARGE_BLOCK` ?

File: Applications/telementry-test/main.cpp [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+    iotBlock.make(block, sizeof(block), 0x132);
+    iotBlock.print();
+    iotBlock.send();
+}
+
+int main()
+{
+    MAIN_INFO("Application started; (id: %d) v%d.%d.%d.%d; vitroio-sdk v%s",
+        VITRIOIO_TEMPLATE_FIRMWARE_ID,
```
Comment (TomaszAIR) [inline Applications/telementry-test/main.cpp:122]: @Stojek139808 can we change FIRMWARE_ID for this app? This way we could have two different apps (IoT-Blocks, Telemetry) and test the Shard OTA (so update with different FW_ID should not be applied to Shards)

**Review by TomaszAIR — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (Stojek139808) [reply .github/workflows/ci.yml]: Yes, I was convinced it was the other way around. Replaced all occurrences.

File: Applications/telementry-test/main.cpp (hunk 3 above)
Comment (Stojek139808) [reply Applications/telementry-test/main.cpp]: Yes, fixed that.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Stojek139808) [reply .github/workflows/ci.yml]: as below, fixed

File: Applications/telementry-test/main.cpp (hunk 4 above)
Comment (Stojek139808) [reply Applications/telementry-test/main.cpp]: Added new ID. There was only `0` in the header so just added a new define with `1`.

**Review by TomaszAIR — APPROVED** [review]

**Pull request merged**